modules/plugin/jdbc/jdbc-oracle/src/test/java/org/geotools/data/oracle/OracleJNDIDataSourceOnlineTest.java

```diff
@@ -55,6 +55,10 @@ public void testExtraParams() {
         List<String> baseJndiParams = getBaseJNDIParams();
         List<String> jndiParams = getParamKeys(getJNDIStoreFactory());
         jndiParams.removeAll(baseJndiParams);
+        standardParams.remove(
+                OracleNGDataStoreFactory.GET_COLUMN_REMARKS
+                        .key); // not including column remarks parameter for JNDI, direct
+        // configuration is explained in the documentation
```

Comment:
Try to get along with the automatic formatter 

```suggestion
        // not including column remarks parameter for JNDI, direct configuration 
        // is explained in the documentation
        standardParams.remove(OracleNGDataStoreFactory.GET_COLUMN_REMARKS.key); 
```